tasks/models.py

```diff
@@ -0,0 +1,17 @@
+import uuid
+
+from datetime import datetime
+from pony.orm import *
+
+
+AccuracyScore = Database()
+
+
+class Task(AccuracyScore.Entity):
+    tasId = PrimaryKey(str, default=uuid.uuid4().hex)
```

Comment:
Consider using https://docs.ponyorm.com/api_reference.html#cmdoption-arg-default, and notice that default should be a function rather than a constant.